vendor/onethinx/otx18.yaml

```diff
@@ -0,0 +1,116 @@
+name: Onethinx LoRaWAN module
+description: Onethinx dual core module with integrated LoRa and LoRaWAN stack, SX126x radio chips, integrated secure element and an integrated omnidirectional antenna.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '18.02'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0'
+    numeric: 1
+  
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867,
+    # RU864-870
+    profiles:
+      EU863-870:
+        vendorID: 307
+        # Identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: otx18-eu868-profile
+        lorawanCertified: true
+        #codec: otx18-eu868-codec
+      US902-928:
+        id: otx18-us915-profile
+        lorawanCertified: false
+        #codec: otx18-eu868-codec
+
+# Additional radios that this device has (optional)
+# Valid values are: ble, nfc, wifi, cellular.
+additionalRadios:
+  - ble
+
+# Dimensions in mm (optional)
+# Use width, height, length and/or diameter
+dimensions:
+  width: 20
+  length: 25
+  height: 3
+
+# Weight in grams (optional)
+weight: 2
+
+# Operating conditions (optional)
+operatingConditions:
+  # Temperature (Celcius)
+  temperature:
+    min: -40
+    max: 85
+  # Relative humidity (fraction of 1)
+  relativeHumidity:
+    min: 0
+    max: 0.95
+
+# Key provisioning (optional)
+# Valid values are: custom (user can configure keys), join server and manifest.
+keyProvisioning:
+  - custom
+  - join server
+  - manifest
+
+# Key programming (optional)
+# Valid values are: bluetooth, nfc, wifi, serial (when the user has a serial interface to set the keys)
+# and firmware (when the user should change the firmware to set the keys).
+keyProgramming:
+  - serial
+  - firmware
+  - bluetooth
+
+# Key security (optional)
+# Valid values are: none, read protected and secure element.
+keySecurity: secure element
+
+# Firmware programming (optional)
+# Valid values are: serial (when the user has a serial interface to update the firmware), fuota lorawan (when the device
+# supports LoRaWAN FUOTA via standard interfaces) and fuota other (other wireless update mechanism).
+firmwareProgramming:
+  - serial
+
+# Product and data sheet URLs (optional)
+productURL: https://www.onethinx.com/module.html
+dataSheetURL: https://onethinx.github.io/Onethinx_Documentation/DoxyDocumentation/html/intro.html
+
+# Commercial information
+resellerURLs:
```

Comment:
in this case are resellers, region can't be World Wide

Reply:
Currently modules can only be bought through us, and we are not "resellers" so I will delete this whole part, thanks!